src/actions/AddSiteActions.js

```diff
@@ -106,8 +95,15 @@ export const checkIfSiteIsReadyForUpload = () => {
     }
 };
 
+export const siteDetailCheckboxWasClicked = ({siteDetailCheckboxKey}) => {
+    return{
+        type: SITE_DETAIL_CHECKBOX_UPDATED,
+        payload: {siteDetailCheckboxKey}
+    }
+};
```

Comment:
New action for when we check a box for site details (`features` or `facilities`)